test/libsolidity/syntaxTests/inheritance/override/nonintermediate_common_base_and_unique_implementation.sol

```diff
@@ -0,0 +1,19 @@
+interface I {
+	function f() external;
+	function g() external;
+}
+interface J {
+	function f() external;
+}
+abstract contract IJ is I, J {
+	function f() external virtual override (I, J);
+}
+abstract contract A is IJ
+{
+	function f() external override {}
+}
+abstract contract B is IJ
+{
+	function g() external override {}
+}
+contract C is A, B {}
```

Comment:
Is this fine because the `B` path doesn't lead to an implementation?

Reply:
For ``f``, yes.

This is fine, since there is a common base interface ``IJ`` and if you look at all the paths from ``C`` to ``IJ`` you will find unique implementations for ``f`` and ``g``.